main.tf

```diff
@@ -139,6 +140,14 @@ resource "azurerm_key_vault_key" "kvkey" {
     "verify",
     "wrapKey",
   ]
+  rotation_policy {
```

Comment:
Can we make is dynamic?

Reply:
Yes, we can set it dynamic but automatic key rotation policy is better to use to rotate the key at specific time interval.

Reply:
Yes, we can set it dynamic but automatic key rotation policy is better to use to rotate the key at specific time interval.

Reply:
By default we can set it to true

Reply:
what?